client/contracts/truffle/test/kns/KNSFunctionality.test.js

```diff
@@ -23,15 +23,15 @@ contract('KNS Functionality', (accounts) => {
 
   it('should register a validator domain', async () => {
     // when
-    await this.registrar.register(web3.sha3('validator'), accounts[0], { from: accounts[0] });
+    await this.registrar.register(web3.sha3('validator'), accounts[0], { from: accounts[1] });
```

Comment:
web3.sha3('validator'), 'validator', 'validator.kowala' - We should define constants because we're using the same values in multiple places.